### DISCUSSION TITLE
How to know implemented VSA-DSS version

### DISCUSSION THREAD

**Discussion by decheu** [QGEP/QGEP#721] [Q&A] [answered]
How can I know what VSA-DSS version is implemented in the (latest) [QGEP datamodel](https://github.com/QGEP/datamodel/releases/)?
E. g. at the moment: Is it the [most recent one](https://vsa.ch/models/2020/VSA_DSS_2020_2_d_LV95-20210625.ili) ?

**Comment by sjib** [comment, marked as answer]
The QGEP datamodel bases on VSA-DSS 2015 ([version 1.5.4 and older](https://github.com/QGEP/datamodel/releases/tag/1.5.4)). There are plans for this year to implement VSA-DSS 2020.

**Reply by decheu** [reply]
Thanks for the specific answer for model version 1.5.4 and older.
Is there / could there be a tag/mini description with each QGEP model release which exact VSA-DSS model it corresponds to? E. g. https://vsa.ch/models/2020/VSA_DSS_2020_2_d_LV95-20210625.ili
What needs to be done and how much effort is it to implement VSA-DSS 2020?

**Reply by sjib** [reply]
You can already export as VSA-DSS 2015 and then use CHECKVSA to convert from 2015 -> 2020. Thats the short term solution.

**Reply by sjib** [reply]
For the release change of TEKSI wastewater / QGEP we need more considerations to cover all the effects on the existing version, export, import, data model upgrade process, integrating the new model extension etc. Just give us some time for that.
